Errors/FailedToGetJobFamilies.ts

```diff
@@ -0,0 +1,8 @@
+export class FailedToGetJobFamiliesError implements Error {
```

Comment:
I think this should be extending error instead 

```suggestion
export class FailedToGetJobFamiliesError extends Error {
```